workbench/tests/binary_tests/puppet.test.js

```diff
@@ -236,7 +240,7 @@ test('Run a real invest model', async () => {
   await page.screenshot({ path: `${SCREENSHOT_PREFIX}6-run-canceled.png` });
 }, 240000); // >2x the sum of all the max timeouts within this test
 
-test('Check local userguide links', async () => {
+test.only('Check local userguide links', async () => {
```

Comment:
Ahh, sorry, I committed this `.only` by accident! @emlys 
```suggestion
test('Check local userguide links', async () => {
```